package.json

```diff
@@ -50,13 +52,15 @@
     "test:e2e:run": "pnpm test:e2e --ui",
     "test:integration:run": "PW_MODE=integration playwright test tests/integration",
     "test:integration": "PW_MODE=integration playwright test --ui tests/integration",
-    "lint": "prettier --check --plugin-search-dir=. . && eslint --ignore-path .gitignore . && stylelint \"src/**/*.{css,postcss,svelte}\"",
-    "lint:fix": "eslint --ignore-path .gitignore --fix",
-    "format": "prettier --write --plugin-search-dir=. . && stylelint \"src/**/*.{css,postcss,svelte}\" --fix",
+    "lint": "pnpm prettier; pnpm eslint; pnpm stylelint",
+    "lint:ci": "pnpm prettier && pnpm eslint && pnpm stylelint",
+    "format": "pnpm prettier:fix; pnpm eslint:fix; pnpm stylelint:fix",
```

Comment:
using `;` in `npm lint` (which should be run locally) will ensure all three are still run even if one fails. Using `&&` in `lint:ci` will fail our lint GitHub action if one of the individual scripts fails.